src/main/java/com/facebook/openwifirrm/optimizers/tpc/RandomTxPowerInitializer.java

```diff
@@ -11,6 +11,7 @@
 import java.util.Map;
 import java.util.Random;
 import java.util.TreeMap;
+import java.util.LinkedList;
```

Comment:
It isn't used anywhere.

@elludraon Is there a way to auto delete unused imports?